### PR TITLE
docs: update import-export app documentation

### PR DESCRIPTION
**Pull request by kabaros**

### CODE REVIEW

**Review by tomzemp — APPROVED** [review]
LGTM

File: src/user/importexport-app.md [hunk 1]
```diff
@@ -251,18 +250,16 @@ recommended format and can also be used to import associated geometries
...
 
 ![](resources/images/import_export/tei_import.png)
 
-1.  Choose a file to upload
+1.  Choose a JSON file to upload
 
-2.  Select a format: _JSON_ or _XML_
-
-3.  Select the appropriate settings for:
+1.  Select the appropriate settings for:
```
Comment (teleivo) [inline src/user/importexport-app.md:262]: will the numbers show correctly when rendered? they are all 1.

File: src/user/importexport-app.md [hunk 2]
```diff
@@ -251,18 +250,16 @@ recommended format and can also be used to import associated geometries
 2.  Click on the **Import** button which will upload the file and start the
     importing process.
 
-### Tracked Entity Instances Import { #tei_import }
+### Tracked Entities Import { #tei_import }
 
-Tracked Entity Instances Import can be accessed from the sidebar by
-clicking on TEI Import.
+Tracked Entities Import can be accessed from the sidebar by
```
Comment (teleivo) [inline src/user/importexport-app.md]: I think we should write proper sentences with https://www.grammarly.com/blog/capitalization-rules.

File: src/user/importexport-app.md (hunk 1 above)
Comment (kabaros) [reply src/user/importexport-app.md:262]: yes these will be autonumbered when rendered

File: src/user/importexport-app.md (hunk 2 above)
Comment (kabaros) [reply src/user/importexport-app.md]: obviously the intention is not to write improper sentences .. I've run this through Joe, and he pointed me to [our guidelines](https://ui.dhis2.nu/patterns/writing#capitalization). 

If there is a mistake, then it's just that .. a mistake. Please point it out and I will fix it.

Comment (kabaros) [reply src/user/importexport-app.md]: I fixed these two [here](https://github.com/dhis2/dhis2-docs/compare/c0636c1562a42d8d80d7a992fe5014dd6d91f71a..1aad4178957070e25ea4f5c4d2615d1e6503f768)

File: src/user/importexport-app.md [hunk 3]
```diff
@@ -399,56 +400,90 @@ You can export event or tracker data in JSON, CSV, or XML formats.
...
+
+The import/export app was upgraded to use the new tracker API for importing and exporting tracked entities and events. The old API will be deprecated in v42 hence all applications are advised to upgrade as soon as possible.
+
+This brings a lot of benefits in terms of the consistency and reliability of the import/export experience for tracked entities and events, such as better validation, error reporting and the use of a more reliable job scheduling workflow.
+
+These benefits come with a couple of important caveats. First, the new format of the exported files is incompatible with previous versions of the app, so exports from v41 can not be imported in previous versions, and exports from previous versions can not be imported in v41. We also dropped support for XML format, and only support JSON and CSV. More context for these changes can be found in the [Tracker API documentation](https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-master/tracker.html).
+
+![Improved error reports in v41+](resources/images/import_export/v41_error_reports.png)
+
+
```
Comment (kabaros) [inline src/user/importexport-app.md]: @teleivo @Bekkalizer this bit here specifically is that I would like a second look from you about - does it sound ok? is there anything more that I should add about the tracker upgrade that's relevant from an end-user perspective

File: src/user/importexport-app.md (hunk 2 above)
Comment (teleivo) [reply src/user/importexport-app.md]: I did not know this guide but assume its for UI only and not docs. I find this

> There are some terms in DHIS2, like organisation unit, that have adopted other spellings. In those cases, use the DHIS2 established spelling.

leads to odd sentences in documentation.

File: src/user/importexport-app.md [hunk 4]
```diff
@@ -399,56 +400,90 @@ You can export event or tracker data in JSON, CSV, or XML formats.
...
     - ID scheme
 
-5.  Click **Export tracked entity instances** which will open a new
+5.  Click **Export tracked entity** which will open a new
     web-browser window that will give you a file to download to your
     local computer.
 
+## Changes to Tracker import/export in v41+ { #v41_tracker_changes }
+
+The import/export app was upgraded to use the new tracker API for importing and exporting tracked entities and events. The old API will be deprecated in v42 hence all applications are advised to upgrade as soon as possible.
```
Comment (teleivo) [inline src/user/importexport-app.md]:
```suggestion
The import/export app was upgraded to use the [new tracker API](https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-master/tracker.html) for importing and exporting tracked entities and events. The [deprecated tracker API](https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-master/tracker.html) will be removed in v42 hence all applications are advised to upgrade as soon as possible.
```

File: src/user/importexport-app.md [hunk 5]
```diff
@@ -399,56 +400,90 @@ You can export event or tracker data in JSON, CSV, or XML formats.
...
+      unit itself.
+
+1.  Decide whether you want to filter by _program_ or _tracked entity type_.
 
-3.  Select a format: _JSON_, _CSV_, or _XML_.
+1. Decide what statuses to include in the export
+
+1. Decide which follow-up statuses to include in the export
+
+3.  Select a format: _JSON_ or _CSV_.
```
Comment (teleivo) [inline src/user/importexport-app.md]: Either keep the numbers in order or make them all 1 😅

**Review by teleivo — APPROVED** [review]

**Pull request merged**